client-src/elements/chromedash-admin-blink-component-listing.js

```diff
@@ -0,0 +1,252 @@
+import {html, css, LitElement} from 'lit';
+import {SHARED_STYLES} from '../sass/shared-css.js';
+import {VARS} from '../sass/_vars-css.js';
+import {LAYOUT_CSS} from '../sass/_layout-css.js';
+
+export class ChromedashAdminBlinkComponentListing extends LitElement {
+  static get styles() {
+    return [
+      SHARED_STYLES,
+      VARS,
+      LAYOUT_CSS,
+      css`
+      :host {
+        display: flex;
+      }
+
+      :host([editing]) .owners_list_add_remove {
+          opacity: 1;
+          pointer-events: all
```

Comment:
Style: Include trailing semicolons to make the file easier to edit and make future diffs more clear.
https://google.github.io/styleguide/htmlcssguide.html#Declaration_Stops

Reply:
Will do!